tsconfig.karma.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "@ethereumjs/config-typescript",
+  "include": ["bin", "lib", "test"],
+  "compilerOptions": {
+    "typeRoots": [
+      "node_modules/@polkadot/ts"
+    ],
+    "skipLibCheck": true
```

Comment:
Have added this new config because karma-typescript is having difficulty resolving some `@types/chrome` libs that are dependencies of `@polkadot/ts`. 

[skipLibCheck][1] disables type checking for third party declaration files. Don't think this option is great to have on for regular test runs but it silences the error for the browser tests...

[1]: https://dd.engineering/blog/typescript-the-skiplibcheck-option-explained